timefold-solver-python-core/src/main/python/score/_constraint_stream.py

```diff
@@ -454,6 +454,20 @@ def concat(self, other):
         else:
             raise RuntimeError(f'Unhandled constraint stream type {type(other)}.')
 
+    def complement(self, item_type: Type[A]) -> 'UniConstraintStream[A]':
```

Comment:
If I'm forced to add this for new methods in constraint streams, why am I not forced to add the new methods on constraint verifier?